cmd/helm/version.go

```diff
@@ -59,8 +59,10 @@ func newVersionCmd(out io.Writer) *cobra.Command {
 		},
 	}
 	f := cmd.Flags()
+	f.BoolP("client-only", "c", true, "display client version information")
```

Comment:
Isn't the flag in helm v2 named `client`?

Reply:
`--client-only` did feel familiar but I couldn't quite figure out why.
Then I searched the code and found that it is `helm init --client-only` but `helm version --client` 😉

Reply:
Gah. I'm mixing flags again -_-

Thanks for catching this.